### ISSUE TITLE
Failed to auto-load MSL after fresh install

### ISSUE THREAD

**Issue by Arinomo** [OpenModelica/OpenModelica#8873]
### Description

I re-installed OM and every time i open OMEdit, it failed to load MSL automatically

![Capture](https://user-images.githubusercontent.com/52284681/165634041-f3536743-c7cc-4200-9a7c-24116900c464.JPG)

```
[1] 23:20:11 Scripting Notification
You can install the requested package using one of the commands:
  installPackage(Modelica, "4.0.0+maint.om", exactMatch=true)
  installPackage(Modelica, "4.0.0", exactMatch=false)
  installPackage(Modelica, "4.0.0", exactMatch=false).

[2] 23:20:11 Scripting Error
Failed to load package Modelica (4.0.0) using MODELICAPATH C:/OpenModelica/lib/omlibrary;C:/Users/UserName/AppData/Roaming/.openmodelica/libraries/.
```

### Steps to Reproduce

Install current nightly build (22/4/2022)
Open OMEdit

### Expected Behavior

MSL is found in installation dir and automatically loaded

### Version and OS

 - OpenModelica Version: v1.20.0-dev-13-g85ae74ca89 (64-bit)
 - OS: Windows 10, 64 bit
 - MSL 4.0.0

**Comment by casella** [comment]
Yes, we already got reports of this issue from @rfranke and @AndreaBartolini, we'll fix it for 1.19.0-beta.2 ASAP.

**Comment by casella** [comment]
This should be the same issue as #8881. @Arinomo please check once that is fixed.

**Comment by Arinomo** [comment]
Similar to issue #8882

**Comment by casella** [comment]
@Arinomo, this is now fixed in the latest 1.20.0 nightly build. As soon as 1.19.0-beta.2 is released, we'll test it there as well and then close the ticket

**Comment by casella** [comment]
Successfully tested in 1.19.0-dev.beta3.
